models/mediaFiles.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+    name: "mediaFiles",
+    schema: {
+        name: {
+          type: String,
+          required: true,
+          index: true
+        },
+        type: {
```

Comment:
@deepakr123 - Put type, Unicode, and status in an index.

Reply:
resolved